src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5936,6 +5937,9 @@ void MusicXMLParserPass2::harmony(const QString& partId, Measure* measure, const
     ha->render();
 
     ha->setVisible(printObject);
+    if (placement == "below") {
+        ha->setPlacement(PlacementV::BELOW);
```

Comment:
Would `ha->setPropertyFlags(Pid::PLACEMENT, PropertyFlags::UNSTYLED);` be needed?

Reply:
I believe it is, IIRC I tried and it didn't work without

Reply:
Hmm, hold on, other way round, it does work without, so doesn't seem needed.
Unlike the color settings further down

Reply:
And I tested again: yes it isn't needed